HealthcareSystem/PatientWebApp/wwwroot/html/search_documentation.html

```diff
@@ -82,7 +81,8 @@ <h3 style="margin-top: 30px; margin-left:20px;">Search documentation</h3>
 
                     <!-- Serch button -->
                     <td width="25%">
-                        <button style=" width:120px; margin-right:30px;" class="btn btn-primary mb-2" id="search_btn" type="submit">Search</button>
+                       <!-- <button style=" width:120px; margin-right:30px;" class="btn btn-primary mb-2" id="search_btn" type="submit">Search</button>-->
+                        <input type="submit" value="Search"/>
                     </td>
                 </tr>
```

Comment:
Will you use this 84th line of code?
My suggestion is to delete it if you won't use it.
And please if it is not a problem, change Serch to Search in comment (I know it was accidentally).